### PR TITLE
Added `open` context menu item

### LINKED ISSUE #753: Missing asset context menu actions
We have 4 different locations for context menu items. The following list shows the currently missing actions:

**1. Tree**
- [x] Download (for single assets)
- [x] Upload new version (for single assets)

**2. Asset folder preview**
Image
- [x] Open
- [x] Download
- Show in tree (we need to do this later)

**3. Asset list view**
Image
=> There the context menu is currently missing totally. We should add it as regular context menu with right click. Maybe we later also add it as 3 dots Button in addition to the right click, but the right click is important as a row can get really long.

- [x] Open
- [x] Rename
- [x] Delete
- [x] Download
- Show in tree (we need to do this later)

This can be exactly the same like in the Asset folder preview. We should share the same code.

**4. Asset detail "more" button:**
Image

For folders:
- [ ] Rename
- [ ] Delete
- [ ] Download as zip

For regular assets:
- [ ] Rename
- [ ] Delete
- [ ] Download
- [ ] Clear thumbnails
- Share via notifications (let's add this later)

### PR DESCRIPTION
**Pull request by Corepex**
## Changes in this pull request
Resolves #753 

## Additional info
This pull request introduces a new feature to the asset editor by adding an "Open" context menu item for assets. The changes involve importing and using the `useOpen` hook, creating the `useOpen` hook, and updating the translations file.

### New Feature: Open Context Menu Item

* [`assets/js/src/core/modules/asset/editor/types/folder/tab-manager/tabs/preview/flex-container.tsx`](diffhunk://#diff-2b470cbd71fbc23030871cdb3b527d570970612c12c782e65943c7358fc4f883R27): Added import for `useOpen`, and included `openContextMenuItem` in the `FlexContainer` component's dropdown items. [[1]](diffhunk://#diff-2b470cbd71fbc23030871cdb3b527d570970612c12c782e65943c7358fc4f883R27) [[2]](diffhunk://#diff-2b470cbd71fbc23030871cdb3b527d570970612c12c782e65943c7358fc4f883R40) [[3]](diffhunk://#diff-2b470cbd71fbc23030871cdb3b527d570970612c12c782e65943c7358fc4f883R53)
* [`assets/js/src/core/modules/element/actions/open/open.tsx`](diffhunk://#diff-12aa2325681f5a0a226035b646a00cf663cabea5e9620890c99896d23c598962R1-R49): Created the `useOpen` hook which provides the `openContextMenuItem` function to generate a context menu item for opening assets.
* [`translations/studio.en.yaml`](diffhunk://#diff-6c9f9f77edb1e43229fb2c23bf06c513915c2f61c6e31b70c95e43d624e8bbbdR326): Added a new translation key `element.open` for the "Open" context menu item.

### CODE REVIEW

File: assets/js/src/core/modules/element/actions/open/open.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+export const useOpen = (elementType: ElementType): UseOpenHookReturn => {
+  const { t } = useTranslation()
+  const { openElement } = useElementHelper()
+
+  const openContextMenuItem = (node: Element): ItemType => {
+    return {
+      label: t('element.open'),
+      key: 'open',
+      icon: <Icon value={ 'union' } />,
+      hidden: !checkElementPermission(node.permissions!, 'rename') || node.isLocked,
```
Comment (markus-moser) [inline assets/js/src/core/modules/element/actions/open/open.tsx]: The permission check needs to be based on the `view` permission and we need to remove the `|| node.isLocked` part of the condition.

**Review by markus-moser — CHANGES_REQUESTED** [review]
Please check my comment, other then that it LGTM and it worked as it should.

File: assets/js/src/core/modules/element/actions/open/open.tsx (hunk 1 above)
Comment (Corepex) [reply assets/js/src/core/modules/element/actions/open/open.tsx]: absolutely correct - corrected it (see: https://github.com/pimcore/studio-ui-bundle/pull/791/commits/e894ae29d0f5ba226d014b5cfa1a471c2922bb3a)

**Review by markus-moser — APPROVED** [review]
thanks, LGTM now

**Pull request merged**